radicle-tracker/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+use std::hash;
+
+pub mod thread;
+use thread::*;
+
+pub struct Issue<Author> {
```

Comment:
What about incremental issue numbers known from many trackers? Or differently what is the identity of an issue, how will it be referred to and referenced?

Reply:
Ya, I think there's another level above `Issue` which is `Tracker` or something. They'll definitely need an identifier.

Reply:
That makes a lot of sense, so `Tracker` likely be the actual collab-tool package.

Reply:
Ya, likely so. I imagine it would be an organisation of open and closed issues. Or something of the like. It'll need more thought :) Getting a single issue done for now :grin: